dev/about.html

```diff
@@ -0,0 +1,89 @@
+	<header>
+		<div class="headerWrapper">
+			<h1>arai miwa</h1>
+			<nav>
+				<a href="index.html">Work</a>
+				<!-- <a href="#">Recource</a> -->
+				<a href="about.html" class="current">About</a>
+			</nav>
+		</div>
+		<div class="header-color"> </div>
+	</header>
+	<main>
+		<div class="contentsWrapper about">
+			<div class="Profile">
+				<div class="profileImage">
+					<img src="assets/images/about-image-cook-mono.jpg" alt="photo" width="100%">
+				</div>
+				<h2 class="contentsTitle">Profile</h2>
+				<p>Arai Miwa<br>1992 年神奈川県生まれ。<br>早稲田大学人間科学部人間環境科学科卒業後、新卒で入社したリノベーション会社にて次期新卒採用や、採用サイト、コーポレートサイトなどのディレクションを担当したことがきっかけでデザインを学び始める。<br>デザイナーとして株式会社チームラボ入社し、アシスタントデザイナーとして従事。<br>社会課題・問題解決としてのデザインに興味を持ち、2016 年から株式会社オハコにUI デザイナーとして入社。<br>主にデジタル領域のサービスの0→1 立ち上げ、改善に関わるデザインに従事している。</p>
+			</div>
+			<div class="Skills">
+				<h2 class="contentsTitle">Skills</h2>
+				<div class="skill">Sketch</div>
+				<div class="skill">Figma</div>
+				<div class="skill">Zeplin</div>
+				<div class="skill">inVision</div>
+				<div class="skill">Principle</div><br>
+				<div class="skill">Illustrator</div>
+				<div class="skill">Photoshop</div>
+				<div class="skill">Animate CC</div>
+				<div class="skill">After Effect</div><br>
+				<div class="skill">HTML5</div>
+				<div class="skill">CSS3</div>
+				<div class="skill">Git</div>
+			</div>
+			<div class="love">
+				<h2 class="contentsTitle">&I Love...</h2>
+				<div class="card3">
+					<div class="card3Thumbnail">
+						<img src="assets/images/about-image-music-mono.jpg" alt="music" width="100%">
+					</div>
+					<h3>Music</h3><a href="https://soundcloud.com/miwa02n/likes" target="blank"><i class="fab fa-soundcloud"></i></a>
+					<p>金管楽器を小学生〜社会人1年目で経験（マーチングバンド全国大会出場。）普段はElectro, Techno, Base, Jazzなど幅広く聞いています。</p>
+				</div>
+				<div class="card3">
+					<div class="card3Thumbnail">
+						<img src="assets/images/about-image-cook-mono.jpg" alt="cook" width="100%">
+					</div>
+					<h3>Cook</h3><a href="https://www.instagram.com/wam01n/" target="blank"><i class="fab fa-instagram"></i></a>
+					<p>夜中にケーキを焼くのが趣味です。小料理屋的な料理も好きです。たまにデリバリー＆主張料理もします。</p>
+				</div>
+				<div class="card3">
+					<div class="card3Thumbnail">
+						<img src="assets/images/about-image-run-mono.jpg" alt="run" width="100%">
+					</div>
+					<h3>Run</h3>
+					<p>2018年秋から体力づくりのために週末ランをはじめました。2019年の目標はフルマラソンを走りきることです。</p>
+				</div>
+			</div>
+		</div>
+	</main>
+	<footer>
+		<div class="footerWrapper">
+			<div class="footerMenu">
+				<a href="index.html">Work</a><!-- 
+				<a href="#">Recource</a> -->
+				<a href="about.html">About</a>
+			</div>
+			<div class="sns">
+				<a href="https://www.instagram.com/wam01n/"><i class="fab fa-instagram" target="blank"></i></a>
+				<a href="https://github.com/miwa02n"><i class="fab fa-github" target="blank"></i></a>
+				<a href="#"><i class="fab fa-twitter" target="blank"></i></a>
+			</div>
+			<p class="copyright">@ AraiMiwa 2019 | Source made by AraiMiwa</p>
```

Comment:
コピーライトを意味する記号は @ (アットマーク) ではありません。
© です(option + g で打てる)
表記すべき内容に従わないと、著作権を有しませんので気をつけましょう。

> https://liginc.co.jp/designer/archives/11313 

記号はコードの記号かコンテンツかを判別するのが難しく、バグの原因になるのでエスケープシーケンス（特殊文字）が用意されいます。なので、そちらを使いましょう。

> https://www.w3schools.com/html/html_entities.asp

```suggestion
			<p class="copyright">&copy; Arai Miwa 2019 | Source made by AraiMiwa</p>
```